source/fundamentals/typescript.txt

```diff
@@ -67,93 +71,50 @@ You can pass a type parameter that extends the ``Document`` interface like this:
       :start-after: start-no-key
       :end-before: end-no-key
 
-Any Type
-~~~~~~~~
+Parameters of Any Type
+~~~~~~~~~~~~~~~~~~~~~~
 
-The following classes accept any type parameter:
+The following classes accept all type parameters that are not recursive:
```

Comment:
Would this be more uniform and therefore easier to read sequentially if it was the same format as the intro sentence in the previous section?
According to the style guide, we should avoid using negatives (e.g. "not") when possible.

Reply:
Updated to:

> The following classes accept all types that both extend
the ``Document`` interface and are recursive:

I think this is an equivalent sentence to:

> The following people sell cars that both drive well and are blue.

I have checked both of these sentences against multiple grammar checkers and cannot find any problems.